docs/features/scripting.rst

```diff
@@ -94,6 +94,24 @@ where:
 * ``command`` and ``args`` are entered exactly like they would be entered by
   a user of your application.
 
+Advanced Support
```

Comment:
This section should go at the end of the file.  As is, there is an extra line that is completely out of place.

Reply:
I don't know what you mean by this. This is already the last section in this file.

Reply:
I pushed a small change to address what I was talking about